tests/testthat/test-layer.r

```diff
@@ -12,6 +12,26 @@ test_that("unknown params create error", {
   expect_error(geom_point(blah = "red"), "Unknown parameters")
 })
 
+test_that("Unknown params create error with validate_params = TRUE", {
+  expect_error(geom_point(blah = "red", validate_params = TRUE),
+               "Unknown parameters")
+})
+
+test_that("Unknown params create warning with validate_params = FALSE", {
+  expect_warning(geom_point(blah = "red", validate_params = FALSE),
```

Comment:
@tdhock Sure. That is certainly doable.
My idea was that while using unknown params:

- If a user doesn't specify `validate_params`, ggplot should throw an error.
- If a user does specify `validate_params = FALSE`, we let them continue with a warning.